java/core/src/test/java/com/redhat/rhn/frontend/dto/test/XccdfTestResultDtoTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2025 SUSE LLC
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ */
+
+package com.redhat.rhn.frontend.dto.test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.redhat.rhn.frontend.dto.XccdfTestResultDto;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+
+class XccdfTestResultDtoTest {
+
+    @Test
+    void testCompletedDate() {
+        XccdfTestResultDto dut = new XccdfTestResultDto();
+
+        Date date1 = new GregorianCalendar(2025, Calendar.DECEMBER, 25).getTime();
```

Comment:
Please prefer  `java.time` classes for new code. `Instant` or `LocalDateTime` are generally a better choice compared to `Calendar`.